lib/pbio/drv/uart/uart_ev3_pru.h

```diff
@@ -11,7 +11,7 @@ void pbdrv_uart_ev3_pru_handle_irq_data(uint8_t line);
 void pbdrv_uart_ev3_pru_set_baudrate(uint8_t line, unsigned int baud);
 
 int pbdrv_uart_ev3_pru_read_bytes(uint8_t line, unsigned char *pdata, int size);
-int pbdrv_uart_ev3_pru_write_bytes(uint8_t line, unsigned char *pdata, int size);
+int pbdrv_uart_ev3_pru_write_bytes(uint8_t line, const unsigned char *pdata, int size);
```

Comment:
```suggestion
int pbdrv_uart_ev3_pru_read_bytes(uint8_t line, uint8_t *pdata, int32_t size);
int pbdrv_uart_ev3_pru_write_bytes(uint8_t line, const uint8_t *pdata, int32_t size);
```

We should make this match the .c file.

Reply:
Happy to do that. I used lowercase for const because I'm referring to the C keyword, which doesn't have a capital letter at the front. But I will add a period at the end, if I can figure out how! :)

Reply:
Gotcha - never mind about this one.

Looking forward to seeing what you're working on.

Reply:
As requested, updated the pru signatures. The .c file signatures were already correct. Also updated the commit message. Please take another look. :)